src/bytes/bytes.go

```diff
@@ -6,30 +6,220 @@
 // It is analogous to the facilities of the strings package.
 package bytes
 
-import (
-	"internal/bytealg"
-	"unicode"
-	"unicode/utf8"
+//+gobra
+
+// @ import (
+// @	. "verification/utils/definitions"
+// @	b "verification/utils/bitwise"
+// @	sl "verification/utils/slices"
+// @	seqs "verification/utils/seqs"
+// @	sets "verification/utils/sets"
+// @ )
+
+// the following is some bookkeeping since we have not (yet) specified
+// "internal/bytealg", "unicode", or "unicode/utf8"
+//
+// each function of the respective package is rewritten to instead
+// call one of the stubs below. Each function follows the convention
+// `package_path__FunctionName`, where `package_path` is the path of the
+// package, with `/` replaced by `_`.
+/* @
+
+// utf8.Codepoints(s) returns the sequence of (utf-8 encoded) codepoints in `s`
+ghost
+pure
+ensures 0 < len(s) ==> 0 < len(res)
+ensures 0 == len(s) ==> 0 == len(res)
+decreases
+func unicode_utf8__Codepoints(s []byte) (res seq[rune])
+
+
+@ */
```

Comment:
definition of ghost members should not be in this file, but instead, on a separate .gobra file